js/index.js

```diff
@@ -0,0 +1,160 @@
+import Book from '../modules/book.js';
+import Contact from '../modules/contact.js';
+
+function createTag(tagName, textContent = null, className = null) {
+  const tag = document.createElement(tagName);
+  tag.textContent = textContent;
+  tag.className = className;
+  return tag;
+}
```

Comment:
- [ ] Please consider refactoring this function to be an arrow function and use ES6 syntax as required in this milestone. 🤓 🔧